lib/galaxy/workflow/modules.py

```diff
@@ -830,42 +836,42 @@ def get_inputs(self):
         cases = []
 
         for param_type in ["text", "integer", "float", "boolean", "color"]:
-            default_source = dict(name="default", label="Default Value", type=param_type)
+            default_source: Dict[str, Union[int, float, bool, str]] = dict(name="default", label="Default Value", type=param_type)
             if param_type == "text":
                 if parameter_type == "text":
-                    default = parameter_def.get("default") or ""
+                    text_default = parameter_def.get("default") or ""
                 else:
-                    default = ""
-                default_source["value"] = default
-                input_default_value = TextToolParameter(None, default_source)
+                    text_default = ""
+                default_source["value"] = text_default
+                input_default_value: Union[TextToolParameter, IntegerToolParameter, FloatToolParameter, BooleanToolParameter, ColorToolParameter] = TextToolParameter(None, default_source)
```

Comment:
Maybe just for legibility move the variable and type declaration right before the if?

Reply:
I can't, as that would require an assignment to either `None` (which would break the type) or a default value (which may not be appropriate here, I don't know the code well enough to judge). However, the code is reformatted now, so maybe that is good enough?